lib/create-response-handler.ts

```diff
@@ -86,11 +88,66 @@ export interface ResponseScenarios {
   onError: (error: Error) => any,
 }
 
+/**
+ * IdentityResponseScenarios
+ *
+ * An object enumerating scenarios that can occur when handling a response from
+ * Verify.
+ *
+ * Relying Parties should handle each of these scenarios appropriately.
+ *
+ * NOTE: Should be used by all new services (not using MSA)
+ */
+export interface IdentityResponseScenarios {
+  /**
+   * Called when handling a success response from an identity provider. Your callback
```

Comment:
```suggestion
   * Called when handling a success response. Your callback
```
I deleted ` from an identity provider` because the response technically comes from GOV.UK Verify Hub. But in any case, the success response only ever comes from one place, right? So then let's keep it simple and not put in the source of the response.

Reply:
`success response` is pretty self-explanatory, but it would be even more rigorous to refer to the scenario by name.